client/lib/utils/time-ago.js

```diff
@@ -0,0 +1,48 @@
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+
```

Comment:
I should have been more clear - I wasn't against the use of `moment()`, I just questioned bringing in all of Calypso's `i18n` module to use it.

Don't feel like you have to remove this now though.